src/javascript/app/ng-wig/ng-wig.component.js

```diff
@@ -51,7 +51,10 @@ angular.module('ngWig')
 
       this.$onInit = () => {
         let placeholder = Boolean(this.placeholder);
-        this.ngModelController.$render = () =>  !placeholder ? $container.html(this.ngModelController.$viewValue || '<p></p>') : $container.empty();
+        this.ngModelController.$render = () => this.ngModelController && this.ngModelController.$viewValue
```

Comment:
Do we need to check for existence of `ngModelController` since it is required from the component beforehand?